resources/views/dashboard/dashboard.blade.php

```diff
@@ -1,26 +1,98 @@
 @extends('layouts.main')
 
 @section('content')
+    <h1>Dashboard</h1>
+    
+    <div class="jumbotron">
+    <h1>Hello, User!</h1>
+    <p>
+        Welcome to E-Learning please click <strong>"Words"</strong> to check the words you have learned and if you 
+      want to take a lesson please click <strong>"Lesson"</strong>
+    </p>
 
-    <div id="wrapper">
-        <!-- Sidebar -->
-        <div id="sidebar-wrapper">
-            <ul class="sidebar-nav">
-                <li class="sidebar-image">
-                    <img src="/prof.png" width="200" height="200" class="img-circle" id="sidebar-image">
-                </li>
-            </ul>
+    <p>
+        <a class="btn btn-primary btn-lg" href="#" role="button">Words</a>
+        <a class="btn btn-success btn-lg" href="#" role="button">Lesson</a>
+    </p>
+</div>
+
+<div class="row">
+    <div class="col-md-4">
+        <div class="panel card red white-text">
+            <p class="card-header">
+                Words Learned
+            </p>
+            <span class="card-result">1000</span>
+        </div>
+    </div>
+    <div class="col-md-4">
+        <div class="panel card blue white-text">
+            <p class="card-header">
+                Number of Users
+            </p>
+            <span class="card-result">259</span>
         </div>
-        <!-- /#sidebar-wrapper -->
+    </div>
+    <div class="col-md-4">
+        <div class="panel card green white-text">
+            <p class="card-header">
+                Number of Categories
+            </p>
+            <span class="card-result">35</span>
+        </div>
+    </div>
+</div>
 
-        <!-- Page Content -->
-        <div id="page-content-wrapper">
-            <div class="container-fluid">
-                <a href="#menu-toggle" class="btn btn-secondary" id="menu-toggle"><i class="glyphicon glyphicon-menu-hamburger"></i></a>
-                <h1>Dashboard</h1>
-                @include('dashboard.content')
+<div class="row">
+    <div class="col-md-6">
+        <div class="panel">
+            <div class="panel-heading">
+                Your Followers
+            </div>
+            <div class="panel-body">
+                <table class="table table-striped">
+                    <tbody>
+                    @for ($i = 0; $i < 10; $i++)
+                        <tr>
+                            <td>
+                                <img src="/prof.png" width="50" height="50" class="img-circle"> Jeremiah Caballero
```

Comment:
@jeremiahC Move all your images to a specific `images` directory, and use `public_path` to access inside public folder